collectd-etcd/README.md

```diff
@@ -0,0 +1,128 @@
+#![](https://github.com/signalfx/integrations/blob/master/collectd-etcd/img/integrations_etcd.png) etcd
+
+_This directory consolidates all the metadata associated with the etcd plugin for collectd. The relevant code for the plugin can be found [here](https://github.com/signalfx/collectd-etcd)_
+
+- [Description](#description)
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [Usage](#usage)
+- [Metrics](#metrics)
+- [License](#license)
+
+### DESCRIPTION
+
+This is the SignalFx etcd plugin. Follow these instructions to install the etcd plugin for collectd.
+
+The [`etcd-collectd`](https://github.com/signalfx/collectd-etcd) plugin collects metrics from etcd members from the three [statistics](https://coreos.com/etcd/docs/latest/v2/api.html#statistics) endpoints and the [metric](https://coreos.com/etcd/docs/latest/v2/metrics.html).
```

Comment:
*nit pick* might say "...from the three statistic endpoints and the metric endpoint" or "... from the three statistics and one metric endpoint".  Something just seems sparse about about the way it is phrased now.